.golangci.yml

```diff
@@ -1,7 +1,7 @@
 # Options for analysis running. Details: https://golangci-lint.run/usage/configuration/#run-configuration
 run:
   timeout: 5m
-  go: "1.19"
```

Comment:
We didn't yet upgrade our core version, right? :thinking:

Reply:
Went back to 1.19